rules/autoconsent/taunton.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "taunton",
+  "vendorUrl": "https://www.taunton.com/",
+  "prehideSelectors": ["#taunton-user-consent__overlay"],
+  "detectCmp": [
+    {
+      "exists": "#taunton-user-consent__overlay"
+    }
+  ],
+  "detectPopup": [
+    {
+      "visible": "#taunton-user-consent__overlay"
```

Comment:
This is always considered 'visible' by `isElementVisible`, causing this rule to run every time. They're setting an `aria-hidden:"true"` attribute though, I wonder if we could use that to figure out visibility?

Reply:
oh, nice catch, thank you! `aria-hidden` works well